### DISCUSSION TITLE
Draft releases don't show assets

### DISCUSSION THREAD

**Discussion by bencefr** [community/community#6331] [Other Feature Feedback, Questions, & Ideas] [answered]
The previous UI gave access to assets on draft releases which is a very useful feature to test these assets. The new UI does not show any assets.

**Comment by MylesBorins** [comment, marked as answer]
Thanks for pointing this out @bencefr. I've opened a bug with engineering to fix this

**Reply by MylesBorins** [reply]
This is fixed now! PTAL
